src/dev/build/tasks/os_packages/docker_generator/run.js

```diff
@@ -29,15 +29,26 @@ const linkAsync = promisify(link);
 const unlinkAsync = promisify(unlink);
 const chmodAsync = promisify(chmod);
 
-export async function runDockerGenerator(config, log, build) {
+export async function runDockerGenerator(config, log, build, ubi = false) {
+  // UBI var config
+  const baseOSImage = ubi ? 'registry.access.redhat.com/ubi7/ubi-minimal:7.7' : 'centos:7';
+  const ubiVersionTag = 'ubi7';
+  const ubiImageFlavor = ubi ? `-${ ubiVersionTag }` : '';
+
+  // General docker var config
   const license = build.isOss() ? 'ASL 2.0' : 'Elastic License';
   const imageFlavor = build.isOss() ? '-oss' : '';
   const imageTag = 'docker.elastic.co/kibana/kibana';
   const versionTag = config.getBuildVersion();
   const artifactTarball = `kibana${ imageFlavor }-${ versionTag }-linux-x86_64.tar.gz`;
   const artifactsDir = config.resolveFromTarget('.');
-  const dockerBuildDir = config.resolveFromRepo('build', 'kibana-docker', build.isOss() ? 'oss' : 'default');
-  const dockerOutputDir = config.resolveFromTarget(`kibana${ imageFlavor }-${ versionTag }-docker.tar.gz`);
+  // That would produce oss, default and default-ubi7
+  const dockerBuildDir = config.resolveFromRepo(
+    'build',
+    'kibana-docker',
+    build.isOss() ? `oss${ ubiImageFlavor }` : `default${ ubiImageFlavor }`
```

Comment:
If this test is for oss, why do we need the ubi image as part of it?

Reply:
well when I wrote this the idea was if in the future we had the need to release an ubi image for oss the only thing we would need to do will be remove the if in the main calling function that is not allowing it. In fact currently is doing nothing. I can remove it

Reply:
I just changed the code and removed it!